index.html

```diff
@@ -1088,6 +1089,257 @@ <h4>Nonce</h4>
       </section>
     </section>
 
+    <section>
+      <h2>The BBS+ Bound Signature Suite 2020</h2>
+
+      <p>
+The BBS+ Bound <a>signature suite</a> 2020 is identical to the BBS+
+<a>signature suite</a> 2020 with the addition of a mechanism for recipient
+binding. It MUST be used in conjunction with the signing and verification
```

Comment:
```suggestion
The BBS+ Bound <a>signature suite</a> 2020 adds a mechanism for recipient
binding, and is otherwise identical, to the BBS+ <a>signature suite</a>
2020. It MUST be used in conjunction with the signing and verification
```